### ISSUE TITLE
Bug: duplicate argument 'name' in path and body

### ISSUE THREAD

**Issue by ASRagab** [yanyongyu/githubkit#16]
Minimal reproduction:
```python
from githubkit import GitHub, TokenAuthStrategy
gh = GitHub(TokenAuthStrategy("")).rest
```

Error:
```
Traceback (most recent call last):

  File "/usr/local/lib/python3.8/dist-packages/IPython/core/interactiveshell.py", line 3326, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)

  File ["<ipython-input-3-4ec18c08ad11>"](https://localhost:8080/#), line 1, in <module>
    from githubkit import GitHub, TokenAuthStrategy

  File "/usr/local/lib/python3.8/dist-packages/githubkit/__init__.py", line 2, in <module>
    from .github import GitHub as GitHub

  File "/usr/local/lib/python3.8/dist-packages/githubkit/github.py", line 16, in <module>
    from .core import GitHubCore

  File "/usr/local/lib/python3.8/dist-packages/githubkit/core.py", line 24, in <module>
    from .auth import BaseAuthStrategy, TokenAuthStrategy, UnauthAuthStrategy

  File "/usr/local/lib/python3.8/dist-packages/githubkit/auth/__init__.py", line 1, in <module>
    from .app import AppAuthStrategy as AppAuthStrategy

  File "/usr/local/lib/python3.8/dist-packages/githubkit/auth/app.py", line 10, in <module>
    from githubkit.rest import (

  File "/usr/local/lib/python3.8/dist-packages/githubkit/rest/__init__.py", line 25, in <module>
    from .actions import ActionsClient

  File "/usr/local/lib/python3.8/dist-packages/githubkit/rest/actions.py", line 2189
    def update_org_variable(
    ^
SyntaxError: duplicate argument 'name' in function definition
```

Not a python expert, but the error seems to be here:
https://github.com/yanyongyu/githubkit/blob/2285d0176a0d01556ced839b275b5ec90a68ab5b/githubkit/rest/actions.py#L2195

https://github.com/yanyongyu/githubkit/blob/2285d0176a0d01556ced839b275b5ec90a68ab5b/githubkit/rest/actions.py#L2238

Same possible issue in the async version of the signature. Issue is not present in 0.9.4

**Comment by yanyongyu** [comment]
It seems both body and path has a field names `name`.

**Comment by yanyongyu** [comment]
I just published a new version and yanked version `0.9.5`. Thx for your feedback! 😄

**Comment by ASRagab** [comment]
Thank you!!
